content/en-us/reference/engine/classes/AudioLimiter.yaml

```diff
@@ -47,8 +53,9 @@ properties:
       - Audio
     writeCapabilities: []
   - name: AudioLimiter.Release
-    summary: ''
-    description: ''
+    summary: 'The amount of time it takes to unclamp previously-loud streams.'
+    description: |
+      The amount of time, in seconds, that it takes for the limiter to unclamp any previously (but not currently) loud streams.
```

Comment:
We could add a sentence giving the range for this parameter

Reply:
Good call
```suggestion
      The amount of time, in seconds, that it takes for the limiter to unclamp any previously (but not currently) loud streams.
      This value ranges from 0.001 to 1.
```